huaweicloud/services/cci/resource_huaweicloud_cciv2_pool_binding.go

```diff
@@ -17,21 +17,26 @@ import (
 	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
 )
 
-var poolBindingNonUpdatableParams = []string{"name", "namespace"}
+var poolBindingNonUpdatableParams = []string{
+	"name", "namespace", "annotations", "labels", "finalizers", "generate_name",
+	"owner_references",
+	"pool_ref", "pool_ref.*.id",
+	"target_ref", "target_ref.*.group", "target_ref.*.kind,name", "target_ref.*.namespace", "target_ref.*.port",
+	"api_version", "kind",
+}
```

Comment:
NonUpdatable should be added in the docs

Reply:
fixed